db/schema.rb

```diff
@@ -362,6 +362,12 @@
     t.index ["casa_org_id"], name: "index_judges_on_casa_org_id"
   end
 
+  create_table "jwt_denylist", force: :cascade do |t|
+    t.string "jti", null: false
+    t.datetime "exp", null: false
+    t.index ["jti"], name: "index_jwt_denylist_on_jti"
+  end
```

Comment:
What is this being used for?

Reply:
This table was used in the past to store "used" jwt's when devise-jwt gem was used for authentication. Not using devise-jwt anymore, so we can remove this table.